src/views/room/Room.vue

```diff
@@ -61,6 +61,16 @@
         show-expand
         :single-expand="true"
       >
+        <template v-slot:item.summary="{ item }">
+          <a
+            :id="`kaltura-media-space-${item.id}`"
+            :href="`${$config.kalturaMediaSpaceUrl}/recscheduling/index/edit-event/eventid/${item.id}`"
```

Comment:
@johncrossman `recscheduling` typo?

Reply:
@paulkerschen Not a typo. It's rec-scheduling; a poorly conceived concatenation.